cql3/statements/cf_prop_defs.cc

```diff
@@ -172,11 +174,19 @@ std::optional<utils::UUID> cf_prop_defs::get_id() const {
     return std::nullopt;
 }
 
+bool cf_prop_defs::is_cdc_enabled() const {
```

Comment:
In general we would probably need a way to decide what should be included into CDC (change itself, pre-image, post-image). The question is what's the best way to do it. Should we do a complex configuration like in compaction and topology strategy?

Reply:
I haven't thought this through, but I assumed that because this configuration comes from a CQL command, which is a string, it would most likely be a string. I was worried that if we use a bool now, and want to upgrade to a string later, we will need to deal with upgrade issues, and maybe it's better to use a string right now. On the other hand, maybe we should just call CDC an experimental feature, and not promise any update path yet.

Reply:
The Cassandra syntax is `WITH CDC = true`; we can support an alterantive syntax WITH CDC = property_map`.

The property map would indicate that it's enabled, and whether we log just the primary key, changes, pre/post, and retention options.

Reply:
Will do the property map.